### ISSUE TITLE
Huge number of Particles - Performance Tipps?

### ISSUE THREAD

**Issue by dwerner95** [tkeskita/BVtkNodes#111]
Hello @tkeskita,

I'm currently exploring the possibility of rendering approximately 10 million particles and am considering using Blender as an alternative to Paraview.

In Paraview, my approach to visualising particles involves either point gaussians or points themselves, which is notably efficient. However, this method restricts me from leveraging ray tracing capabilities.

I'm curious if you're aware of any strategies to reduce the mesh count when working with particles. Typically, I would opt for the Sphere-glyph representation, but this leads to an exorbitant triangle count of around 640 million, which is far from ideal.

What are your thoughts on this?

**Comment by tkeskita** [comment]
Try the [VTK to Blender Particles node](https://bvtknodes.readthedocs.io/en/latest/BVTKNodes.html#vtk-to-blender-particles). It allows point instancing. Docs have an example workflow, it should work if you get all the buttons pressed exactly in right order, but it's somehow very sensitive, can break upon any change.

**Comment by dwerner95** [comment]
Youre right, its very sensitive.... thats not much fun to work with, however, it works! And considering the amount pf particles i ask it to render it is also very quick! 

Thanks so muc @tkeskita, this again helps me so much!

Do you think there is any chance we can introduce motion blur into the particles?

**Comment by tkeskita** [comment]
Hi,
question is how to add particle velocity via Python in a Blender particle system so particles get motion blur. I have no idea. I remember several years ago it has been discussed and requested, but I don't know if it is possible. It's probably again a question of finding some Blender add-on which has similar functionality, then we can do the same..

**Comment by dwerner95** [comment]
I think i found something. I am gonna give it a try again, lets see if that works. 
Thanks @tkeskita!
